app/wama.web/Controllers/CheckInController.cs

```diff
@@ -60,6 +71,30 @@ public IActionResult WaiverNotSigned()
             return View();
         }
 
+        [HttpPost]
+        public async Task<IActionResult> Waiver(string signerName, string memberId)
+        {
+            var userAccount = await _UserAccountService.GetUserAccountAsync(memberId);
```

Comment:
That's true. 

It's just a good practice.
When possible, don't blindly trust that everything always work.
https://en.wikipedia.org/wiki/Defensive_programming